src/bundle/Resources/public/scss/_filters.scss

```diff
@@ -31,6 +31,16 @@
             position: relative;
         }
 
+        &--created, &--modified {
+            .ibexa-date-time-picker {
+                width: 100%;
+
+                &__input {
+                    font-size: calculateRem(13px);
```

Comment:
I've decreased font size by 1px in order to avoid overlap between selected range and clear input action.